roles/ovirt-ova-pack/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+  - name: Run packing script
+    script: pack_ova.py "{{ ova_file.dest }}" "{{ ovf }}" "{{ disks }}"
```

Comment:
Can you please document somwhere how to run it? Some example playbook? And also what every variables means?